### ISSUE TITLE
Not maintained any more?

### ISSUE THREAD

**Issue by damomurf** [jenkinsci/mesos-plugin#273]
I've been led to believe that CloudBees are no longer maintaining this plugin as they had kindly been previously.

@Vlatombe - is it possible to get a statement that this is, in fact, the case and maybe get the README in this Repo updated to reflect that and whether there are any plans going forward?

**Comment by sjcollin** [comment]
Yes. that would be great as this plugin hasn't worked since Jenkins 2.27.   Appreciate the response on status of this plugin.

**Comment by maselvaraj** [comment]
@sjcollin I'll fix the plugin for the issue with Jenkins 2.X. @damomurf Can you please let me know what is the issue you are facing ?

**Comment by damomurf** [comment]
I think there are currently 49 issues that folks are currently facing. In particular I've had to invest significant effort in working around things related to #270 and something similar to #268. I've just had a look at the latest Jenkins 2 and noticed that support seems to be non-existent with regard to pipeline builds.

**Comment by Vlatombe** [comment]
@damomurf I maintain 

@damomurf what kind of support are you looking with pipeline? Any cloud plugin should work using the `node` keyword.

**Comment by damomurf** [comment]
@Vlatombe Support for the MesosSingleUseSlave wrapper would be handy. I've tried it but it throws an exception.

**Comment by maselvaraj** [comment]
@damomurf I've added an option to select Single Use Slave default for all the jobs. https://github.com/jenkinsci/mesos-plugin/pull/275. Will this work for you ?

**Comment by maselvaraj** [comment]
Release 0.14.0 is available in update center which includes a fix for using MesosSingleUseSlave with Jenkins pipeline. https://github.com/jenkinsci/mesos-plugin/pull/277

**Comment by MrSoul42** [comment]
@maselvaraj #277 is only at steps level in each stage as far as i can see, if i set a agent for the complete pipeline there is no way i found that you can specify that agent to be singe use. Would it not be beter to use the options mechanism for this:
```
pipeline {
    agent { label 'mesos-agent' }
    options {
        timeout(time: 30, unit: 'MINUTES')
        mesosSingleUseSlave(true)
    }
}
```
The options can also be set at stage level:
```
pipeline {
    agent none
    stages {
        stage('build') {
            agent { label: 'mesos-agent' }
            options {
                mesosSingleUseSlave(true)
            }
        }
    }
}
```
see: https://jenkins.io/doc/book/pipeline/syntax/#options

**Comment by jeschkies** [comment]
@damomurf, I'm the new maintainer. We've recently released 1.0 of the plugin and an experimental 2.0.
